tests/behaviour/util.rs

```diff
@@ -133,7 +134,13 @@ fn value_equals_str(value: &Value, expected: &str) -> bool {
         Value::Boolean(val) => {
             expected.parse::<bool>().and_then(|expected| Ok(expected.eq(val))).unwrap_or_else(|_| false)
         }
-        Value::DateTime(val) => format_datetime(val) == expected,
+        Value::DateTime(val) => {
+            if expected.contains(":") {
+                val == &NaiveDateTime::parse_from_str(expected, "%Y-%m-%dT%H:%M:%S").unwrap()
+            } else {
+                val == &NaiveDateTime::parse_from_str(expected, "%Y-%m-%d").unwrap()
+            }
+        }
```

Comment:
Definitely the right idea in that we should parse the datetime from the scenario, rather than the way this was before. This implementation is a little brittle, but fine for tests.

Reply:
Thank you